build/commands/lib/config.js

```diff
@@ -408,6 +408,21 @@ Config.prototype.buildArgs = function () {
     args.use_vaapi = true
   }
 
+  // Enable Page Graph only in desktop builds.
+  if (this.targetOS !== 'android' && this.targetOS !== 'ios') {
+    args.enable_brave_page_graph = true
+  } else {
+    args.enable_brave_page_graph = false
+  }
+  // Enable Page Graph WebAPI probes only in dev/nightly builds.
+  if (args.enable_brave_page_graph &&
+      (!this.isBraveReleaseBuild() || this.channel === 'dev' ||
+       this.channel === 'nightly')) {
+    args.enable_brave_page_graph_webapi_probes = true
```

Comment:
same as above re config.js vs gn

Reply:
described why I need this explicitly in the previous comment.